node_modules/oae-core/groupprofile/bundles/default.properties

```diff
@@ -0,0 +1,3 @@
+ADD_A_DESCRIPTION_TO_THIS_GROUP = Add a description to this group
+NO_DESCRIPTION_PROVIDED_YET = No description provided yet
```

Comment:
If we decide to remove the 'about' header then yes the profile pics should move up